drivers/sensor/lis2dh/lis2dh.c

```diff
@@ -367,6 +368,12 @@ int lis2dh_init(const struct device *dev)
 			    CONFIG_SENSOR_INIT_PRIORITY,		\
 			    &lis2dh_driver_api);
 
+#define IS_LSM303AGR_DEV(inst) \
+	DT_NODE_HAS_STATUS(DT_INST(inst, st_lsm303agr_accel), okay)
+
+#define DISC_PULL_UP(inst) \
+	IS_ENABLED(DT_INST_PROP(inst, disconnect_sdo_sa0_pull_up))
```

Comment:
Just use DT_INST_PROP; there's no need for IS_ENABLED. A boolean property is 0 or 1 in C depending on if the property is absent or present in DTS.

See the DT_PROP docs: https://docs.zephyrproject.org/latest/reference/devicetree/api.html#c.DT_PROP

Reply:
OK, got it.
Thanks